src/App.tsx

```diff
@@ -1,61 +1,107 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const MY_REGEX = /[^a-z-а-я-0-9-' ']/gi;
+
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [userId, setUserId] = useState<number>(0);
+  const [title, setTitle] = useState<string>('');
+  const [errorTitle, setErrorTitle] = useState<boolean>(false);
+  const [errorUser, setErrorUser] = useState<boolean>(false);
+
+  const addUser = () => {
+    const newId = Math.max(...todos.map(({ id }) => id)) + 1;
```

Comment:
it's bad practice to use latest max id, just create counter with `useState` and increase it

Reply:
I`ll fix it, though it was stated in the readme to make it this way.)